src/style-sheets/modules/_colors.scss

```diff
@@ -3,6 +3,13 @@ $olGreen: #296c37;
 $olDarkGreen: #16341c;
 $olErrorMessage: #FF0000;
 
+//Brand Themes
+$olLTPrimary: #FFFFFF;
+$olLTAccent: #43A850;
+
+$olLTHeader: #262626;
+$olLTCopy: #2B2D31;
+
 // Supporting Colors
 $white: #ffffff;
 $black: #000000;
```

Comment:
I like the variable for all the colors. Could we do that with font-size as well?

Reply:
Yup

Reply:
Done